user/articles/3-layout-and-design/12-widget-reference.markdown

```diff
@@ -3,160 +3,104 @@
 Liferay Commerce ships with a wide range of widgets you'll use to construct your
 site. This section provides a convenient overview in one place.
 
-## Account Management
+* **Account Management**: The Account Management widget allows buyers, sales agents and administrators to
+  manage their accounts without having to access to the Liferay control panel.
 
-The Account Management widget allows buyers, sales agents and administrators to
-manage their accounts without having to access to the Liferay control panel.
+  Administrators can view a list of all accounts, create new accounts, existing
+  accounts, relate accounts to organizations, add members to accounts and manage
+  the roles and profile data of account members. Different roles, e.g. sales
+  agents or buyers, can be given fewer permissions.
 
-Administrators can view a list of all accounts, create new accounts, existing
-accounts, relate accounts to organizations, add members to accounts and manage
-the roles and profile data of account members. Different roles, e.g. sales
-agents or buyers, can be given fewer permissions.
+* **Cart**: The Cart widget displays the cart’s current contents. It includes controls
+  to change the quantity of the products it holds, or to delete them entirely.
 
-## Cart Widgets
+* **Cart Summary**: The Cart Summary widget displays the total price of all items in the cart,
+  including a checkout button. It doesn’t list the cart’s contents.
 
-The **Cart** widget displays the cart’s current contents. It includes controls
-to change the quantity of the products it holds, or to delete them entirely.
+* **Mini Cart**: The Mini Cart displays the cart’s contents and includes a checkout button,
+  but has no controls to edit its content. Instead, it displays an Edit Cart
+  button that links to the Cart widget. Because of this link, the Cart Mini widget
+  can’t function unless a page containing a Cart Content widget is present.
 
-The **Cart Summary** widget displays the total price of all items in the cart,
-including a checkout button. It doesn’t list the cart’s contents.
+* **Category Content**: The Category Content widget allows you to display the products contained in a category and all the related products and content.
```

Comment:
I still think it's getting caught up in the details of how the widgets work and not clearly communicating what you use them to do. What do you think of reworking to be something like this? Do you think it's an over-simplication?

Reply:
P.s. this is a suggestion for replacing the first sentence of the description. If there's agreement on what that first sentence should say, then I still think there's value in you expanding on the description in 1-2 more sentences.